docs/leo-rover/manuals/boot-and-firmware.mdx

```diff
@@ -19,35 +16,51 @@ Prepare:
   height="802"
 />
 
-To connect to Leo Rover, see:
+---
 
-<LinkButton docId="leo-rover/guides/connect-to-rover-ap" />
+## Step 2
 
-:::note
+<h3>Connect to Leo Rover by following this tutorial:</h3>
 
-This time you won't see the battery LED flashing. It will work only after
-successfully performed Step 2.
+## <LinkButton docId="leo-rover/guides/connect-to-rover-ap" />
 
-:::
+---
 
-## Step 2: Flash the firmware
+## Step 3
 
-Refer to:
+<h3>Flash the new firmware by referring to:</h3>
 
-<LinkButton docId="leo-rover/guides/firmware-update" />
+## <LinkButton docId="leo-rover/guides/firmware-update" />
```

Comment:
Do not use headers as means of highlighting text. Use `<span>` instead. Also, do not highlight `<LinkButton>` components.


```suggestion
## Step 2

<span style={{ fontSize: '1.2em', fontWeight: 'bold' }}>
  Connect to Leo Rover by following this tutorial:
</span>

<LinkButton docId="leo-rover/guides/connect-to-rover-ap" />

---

## Step 3

<span style={{ fontSize: '1.2em', fontWeight: 'bold' }}>
  Flash the new firmware by referring to:
</span>

<LinkButton docId="leo-rover/guides/firmware-update" />

```

Reply:
It is confusing for indexing bots from Google etc.

Reply:
Done